examples/stock_server.rs

```diff
@@ -6,7 +6,7 @@ use zeromq::*;
 
 #[async_helpers::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let mut rng = rand::thread_rng();
+    let mut rng = rand::rng();
```

Comment:
Since I wasn't familiar with what was deprecated I'm going to go through each of these and mark with a 👍